src/main/java/clover/AddDeadlineCommand.java

```diff
@@ -0,0 +1,27 @@
+package clover;
+
+class AddDeadlineCommand extends Command {
+    private final String arg;
+    public AddDeadlineCommand(String arg) {
+        this.arg = arg;
+    }
+
+    @Override void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        if (arg == null || arg.trim().isEmpty()) throw new DukeException("Missing task description!!");
```

Comment:
Conditionals should be put on a separate line and single statement conditionals should be wrapped by curly brackets.

```suggestion
        if (arg == null || arg.trim().isEmpty()) {
            throw new DukeException("Missing task description!!");
        }
```